SuperCalculator/src/supercalculator/CalculatorUtils.java

```diff
@@ -0,0 +1,46 @@
+package supercalculator;
+
+import java.util.Scanner;
+
+public class CalculatorUtils {
+
+    /**
+     * Gets standard function to prompt the user for rounding decimal across all functions and prints result, whith error check
+     *@param result  final result that a function produces
+     */
+    public  static void getOutput(double result)
+    {
+        while(true) {
+
+            System.out.println("Would you like to round the result to a given number of Decimal" +
+                    " places? please enter the number or (n) for No value must be > 0");
+            Scanner scannerInput = new Scanner(System.in);
+            String choice = scannerInput.nextLine();
+
+            if (choice.equalsIgnoreCase("n")) {
+                break;
+            }
+            else
+            {
+                try {
+                    if (Integer.parseInt(choice) < 0  || choice.matches("^\\d+\\.\\d+") ) {
+                        System.err.println("Please enter a number > 0 and not a decimal number");
```

Comment:
I think we can simply say, Please enter an Integer bigger than 0.  What do you think?